### PR TITLE
add ci from branches/ci-setup to clean branch

### LINKED ISSUE #1: CI/CD Setup
Set up CI/CD with Github actions. There's a pretty good guide here for reference https://github.com/OfficeDev/TeamsFx/blob/main/docs/cicd_insider/README.md

### PR DESCRIPTION
**Pull request by rorochaudhary**
Changed the triggers to be for all PR requests and all commits to master and removed extraneous commits from branch ci-setup

### CODE REVIEW

File: .github/workflows/github-ci-beaverhours.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      # Currently, no opinionated solution for unit test provided during scaffolding, so,
+      # set up any unit test framework you prefer (for example, mocha or jest) and update the commands accordingly in below.
+      # - name: Run Unit Test
+        # run: cd bot && npm run test && cd -
+ 
+      # # Tab integration tests are off as tab development is a stretch goal.
+      # # ----- Build and test tab-----
+      # - name: Build the project
+      #   # run: cd tabs && npm ci && npm run build && cd -
+      #   run: cd tabs && npm install & npm run build && cd -
```
Comment (richardcoucoules) [inline .github/workflows/github-ci-beaverhours.yml]: You can delete all of this commented code, as we'll be able to add it back when we need to implement it.

**Review by richardcoucoules — CHANGES_REQUESTED** [review]
Hi Rohit,

This looks like a great start! It looks like the CI pipeline is failing as you've added a Prettier check but haven't Prettified the code. A quick fix for this would be running `npx prettier --write .` from the base directory and committing the results. Ideally, you would want to install Prettier as a dev dependency (`npm i -D prettier`) from the base directory and also install the Prettier extension in VSCode so you can get Prettify-on-save functionality in your IDE.

Let me know if you have any problems with any of these steps!

Richard

File: .github/workflows/github-ci-beaverhours.yml (hunk 1 above)
Comment (rorochaudhary) [reply .github/workflows/github-ci-beaverhours.yml]: Removed the tab testing code and formatted the project files using Prettier as suggested. Good call on the vscode extension. Let me know if these changes are sufficient!

**Review by richardcoucoules — APPROVED** [review]
Looks great!

**Pull request merged**